ui/app/utils/classes/stream-logger.js

```diff
@@ -32,7 +32,17 @@ export default EmberObject.extend(AbstractLogger, {
     let buffer = '';
 
     const decoder = new TextDecoder();
```

Comment:
I know you didn't change these here but noticed you could move these underneath the !reader return because you don't use them until after that (both `buffer` and `decoder`)

Reply:
Good call, I'll fix this while I'm in here.